templates/partials/molecule-steps.hbs

```diff
@@ -0,0 +1,8 @@
+<ol class="grav-c-steps">
+  {{#each steps}}
+  <li>
+    <h3>{{title}}</h3>
+    <p>{{description}}</p>
```

Comment:
it might be a good idea to wrap the description in `{{{ }}}` so that you get all html element interpreted in case there are any...